bin/dev/kube-ready-state-check.sh

```diff
@@ -87,7 +87,13 @@ fi
 # kube-dns shows 4/4 ready
 if having_category kube ; then
     kubectl get pods --namespace=kube-system --selector k8s-app=kube-dns | grep -Eq '([0-9])/\1 *Running'
-    status "kube-dns should shows 4/4 ready"
+    status "kube-dns should be running (show 4/4 ready)"
+fi
+
+# tiller-deploy shows 4/4 ready
+if having_category kube ; then
+    kubectl get pods --namespace=kube-system --selector name=tiller | grep -Eq '([0-9])/\1 *Running'
```

Comment:
Will do. I guess the kube-dns check coming before should get the same, I copy-pasta that one.